app/Filters/Conditions/InFilter.php

```diff
@@ -11,6 +11,8 @@ class InFilter implements ConditionContract
 {
     public function __invoke(Arrayable $item, ConditionalsContract $condition): bool
     {
-        return in_array($condition->getComparatorValue(), Arr::get($item->toArray(), $condition->getComparatorField()));
+        $arrayOfData = explode(',', $condition->getComparatorValue());
```

Comment:
There should probably be a change in logic in the frontend to ensure this is properly communicated to the end users.

Reply:
Nahh. I'll do it in another PR.